addons/stock/models/stock_inventory.py

```diff
@@ -153,6 +158,22 @@ def action_reset_product_qty(self):
         self.mapped('line_ids').write({'product_qty': 0})
         return True
 
+    def action_validate(self):
+        track_line_ids = self.line_ids.filtered(lambda x: x.product_id.tracking in ['lot','serial'] and not x.prod_lot_id)
+        if track_line_ids:
+            track_products = track_line_ids.mapped('product_id.product_tmpl_id.name')
+            return {
+                 'name': _('Stock Track Confirmation'),
+                 'type': 'ir.actions.act_window',
+                 'view_type': 'form',
+                 'view_mode': 'form',
+                 'res_model': 'stock.track.confirmaton',
+                 'context': {'default_track_products': ', '.join(track_products), 'default_inventory_id': self.id},
```

Comment:
wouldn't it be better to display in the wizard a one2many with the problematic inventory lines?